### ISSUE TITLE
VoiceOver support

### ISSUE THREAD

**Issue by btzr-io** [btzr-io/godot-aria#18]
There is some partial support for VoiceOver but using `ariaActiveDescendant` doesn't work as well like NVDA or JAWS.
